src/Containers/Info/style.module.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../common/util";
+
+.container {
+  @include m;
+  // text-align: left;
+  // position: absolute;
+  // left: 0;
+  // top: 0;
+  // z-index: 1;
+
+  @include medium {
+    margin-top: var(--page-gutter);
+    margin-right: var(--page-gutter);
+    margin-left: var(--page-gutter);
+  }
+
+}
+
+.nnTitle {
+  font-size: 1.5rem;
+  font-weight: bold;
+}
+
+.infoWrapper h3 {
+  margin-top: 1em;
+}
+
+.infoContainer {
+  column-count: 2;
+  text-align: justify;
+  -webkit-column-width: 350px;
+  -moz-column-width: 350px;
+  column-width: 350px;
```

Comment:
should these be hardcoded?

Reply:
they have their own classes so I'm assuming it won't be an issue with other elements sidewide. Not sure if it's the best way to style with current styling changes. @blerchin thoughts?

Reply:
I think this is correct in this case. We want to set the column width based on the readable width of a column, not as a fraction of a viewport. So I guess strictly speaking it might be better to do it in ems but that is directly equivalent to pixels so it doesn't really matter.